style.css

```diff
@@ -671,14 +799,133 @@ div {
     margin-bottom: 127px;
   }
 
-  .error {
+  /* Popup Window */
+
+  .popup-cards-container {
     display: none;
-    color: rgb(190, 54, 54);
-    width: 37%;
-    margin: 0 auto;
-    position: relative;
-    bottom: 180px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    padding: 5px;
+    z-index: 99999;
+    overflow-y: auto;
+  }
+
+  .popup-cards {
+    border-radius: 16px;
+    border: 1px solid var(--neutrals-n-30-l, #ebecf0);
+    background: var(--white, #fff);
+    box-shadow: 0 48px 48px 0 rgba(37, 47, 137, 0.08);
+    margin: 0 24px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 12px;
+  }
+
+  .open-window {
+    display: block;
+  }
+
+  .popup-cards-title {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .popup-cards-title h2 {
+    color: var(--theme-1-neutrals-1, #091e42);
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    margin-top: 24px;
+    margin-left: 24px;
+  }
+
+  .popup-cards-title img {
+    width: 24px;
+    height: 24px;
+    margin-top: 40px;
+    margin-right: 32px;
+  }
+
+  .popup-cards-details {
+    display: flex;
+    gap: 2%;
+    margin-left: 30px;
+    align-items: flex-end;
+  }
+
+  .popup-cards-details h6:nth-child(1) {
+    color: var(--neutrals-n-600, #344563);
+    font-size: 15px;
+    font-weight: 500;
+    line-height: 20px;
+  }
+
+  .popup-cards-img img {
+    margin-left: 20%;
+    width: 60%;
+  }
+
+  .popup-cards-description {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 111px;
+    margin-right: 32px;
+  }
+
+  .popup-cards-para p {
+    color: var(--neutrals-n-600, #344563);
     font-size: 16px;
+    line-height: 30px;
+    margin-left: 24px;
+    margin-right: 19px;
+    margin-top: 30px;
+  }
+
+  .popup-cards-tech ul {
+    display: flex;
+    gap: 8px;
+    padding-right: 20px;
+    margin-top: 30px;
+  }
+
+  .popup-cards-tech ul li {
+    list-style: none;
+    color: var(--primary-b-400, #6070ff);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    letter-spacing: 0.36px;
+    border-radius: 8px;
+    background: var(--primary-b-50, #ebebff);
+    padding: 4px 12px;
+  }
+
+  .popup-cards-button {
+    display: flex;
+    gap: 10px;
+    margin-top: 20px;
+  }
+
+  .popup-cards-button button {
+    color: var(--primary-b-500, #396df2);
+    text-align: center;
+    font-size: 17px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0.51px;
+    border-radius: 8px;
+    border: 1px solid var(--primary-b-400, #6070ff);
+    padding: 12px;
+    background: #fff;
+    cursor: pointer;
+  }
```

Comment:
Great work on implementing the changes suggested by the previous reviewer to the best of your knowledge. It shows your dedication and effort toward improving the project. However, I noticed that there are still some changes missing from the suggestions provided by the previous reviewer but don't worry I will be celebrating further how to fix this issues 😊:

- [ ] For your design to match that of the `Figma` design and also look good in other to create a good user experience kindly consider adding a `hover` effect on your `buttons`. See below for more details:

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="630" alt="Screenshot 2023-06-28 at 23 18 24" src="https://github.com/78fatemanazari/Portfolio/assets/107192076/95911d94-0ad5-405d-a6d7-2d7c6cfc38ae">
</td>
<td>
<img width="392" alt="Screenshot 2023-06-28 at 23 00 49" src="https://github.com/78fatemanazari/Portfolio/assets/107192076/60abe82a-a68e-4e73-84c8-a5607fa2d265">

</td>
</tr>
</table>

Reply:
Hi dear @Strangeal 
I wanted to work on button interactions but you know when I add a hover or active pseudo-classes, I need to change the icons inside the buttons, but there is no white in those images in the Figma design.  Can you suggest to me how I can implement that? Kindly guide and help me. Thanks

Reply:
Hi @78fatemanazari, 
Sorry for the late reply, You can get these icons from [font-awesome](https://fontawesome.com/) where you can easily change the color of the icons on `hover` by just adding a `color` property.